controllers/UserController.py

```diff
@@ -1,6 +1,12 @@
 from flask import Blueprint, jsonify, request
+import json
+
 from services.UserService import UserService
```

Comment:
Not an issue, but our modules now re-export the individual classes so you can import the desired class directly: 
```suggestion
from services import UserService
```